rust/main/lander/src/adapter/chains/ethereum/nonce/state/tests.rs

```diff
@@ -36,7 +36,7 @@ async fn test_update_nonce_status_pending_inclusion() {
     precursor.tx.set_nonce(nonce);
 
     state
-        .update_nonce_status(&tx, &TransactionStatus::PendingInclusion)
```

Comment:
IMO having `Inclusion` in the name is quite crucial, since it makes the difference between paying enough gas and not doing so; it also closely relates to the "Inclusion Stage" name. `Pending` sounds very ambiguous

Reply:
The main reason to rename is that `PendingInclusion` state transitions into `Mempool` and after that into `Included`. So, there are two stages in "including" transaction: `Pending` and `Mempool`. I am happy to rename in back though.

Reply:
I'd rather remove the Mempool stage but keep PendingInclusion, agree it doesn't add much